thoth/storages/graph/postgres.py

```diff
@@ -3287,14 +3289,30 @@ def get_python_package_versions_all_count(
         if python_version is not None:
             query = query.filter(PythonPackageVersion.python_version == python_version)
 
+        if sort_by:
+            if sort_by == QuerySortTypeEnum.PACKAGE_VERSION.value:
+                query = query.order_by(func.count(
+                            PythonPackageVersion.package_version).desc()
```

Comment:
This is not that simple - it's not just string sorting. We will need to sort based on result of the query (which makes pagination invalid).